### DISCUSSION TITLE
Azure error: [1] 2023-03-14T15:28:42.786Z [AzureAccount] warn: Unable to get estimate data for Azure subscription <subscription-id>: Cannot read properties of undefined (reading 'date')

### DISCUSSION THREAD

**Discussion by stefanatanasiu** [cloud-carbon-footprint/cloud-carbon-footprint#1078] [Q&A] [answered]
Hi Folks,

I am getting the following error when trying to use the app with Azure:

Azure error: [1] 2023-03-14T15:28:42.786Z [AzureAccount] warn: Unable to get estimate data for Azure subscription <subscription-id>: Cannot read properties of undefined (reading 'date')

I have tried with different client ids in different tenants. I am confident I have setup the permissions correctly, but I still get the error. Any thoughts around how to fix ?

Thanks :)

Stefan

**Comment by 4upz** [comment, marked as answer]
Hi @stefanatanasiu! Thanks for trying out CCF and reporting this error. This doesn't seem like a permissions error, but rather an issue reading a property for that row within the consumption data for that subscription. 

Since we've received a similar report in another forum, we've made an issue on our project board to prioritize a fix before the next release and will share updates on it here #1082

**Reply by 4upz** [reply]
Just wanted to follow up and say that this issue has been resolved and released with Create-App 2.3.3. Please consider creating a new app using `npx @cloud-carbon-footprint/create-app@latest` or making the following changes and doing another yarn install as shown in this commit: https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/commit/8ba1fc5cdb77c47c595e36280fae935a57ebaf09

You can read more details behind the issue in #1082
